modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/view.jsp

```diff
@@ -227,52 +232,36 @@ KBArticleURLHelper kbArticleURLHelper = new KBArticleURLHelper(renderRequest, re
 										</h2>
 
 										<span class="text-default">
-
-											<%
-											Date modifiedDate = kbArticle.getModifiedDate();
-
-											String modifiedDateDescription = LanguageUtil.getTimeDescription(request, System.currentTimeMillis() - modifiedDate.getTime(), true);
-											%>
-
-											<liferay-ui:message arguments="<%= new String[] {HtmlUtil.escape(kbArticle.getUserName()), modifiedDateDescription} %>" key="x-modified-x-ago" />
+											<liferay-ui:message arguments="<%= new String[] {HtmlUtil.escape(kbArticle.getUserName()), kbArticleViewDisplayContext.getModifiedDateDescription(kbArticle)} %>" key="x-modified-x-ago" />
 										</span>
 
-										<%
-										int childKBArticlesCount = KBArticleServiceUtil.getKBArticlesCount(scopeGroupId, kbArticle.getResourcePrimKey(), WorkflowConstants.STATUS_ANY);
-										%>
-
-										<c:if test="<%= childKBArticlesCount > 0 %>">
-											<liferay-portlet:renderURL varImpl="childKBArticlesURL">
-												<portlet:param name="mvcPath" value="/admin/view_kb_articles.jsp" />
-												<portlet:param name="redirect" value="<%= currentURL %>" />
-												<portlet:param name="resourcePrimKey" value="<%= String.valueOf(kbArticle.getResourcePrimKey()) %>" />
-												<portlet:param name="resourceClassNameId" value="<%= String.valueOf(kbArticle.getClassNameId()) %>" />
-												<portlet:param name="parentResourceClassNameId" value="<%= String.valueOf(kbArticle.getClassNameId()) %>" />
-												<portlet:param name="parentResourcePrimKey" value="<%= String.valueOf(kbArticle.getResourcePrimKey()) %>" />
-												<portlet:param name="selectedItemId" value="<%= String.valueOf(kbArticle.getResourcePrimKey()) %>" />
-											</liferay-portlet:renderURL>
-
+										<c:if test="<%= kbArticleViewDisplayContext.getChildKBArticlesCount(scopeGroupId, kbArticle) > 0 %>">
 											<span class="text-default">
-												<aui:a href="<%= childKBArticlesURL.toString() %>">
-													<liferay-ui:message arguments="<%= childKBArticlesCount %>" key="x-child-articles" />
+												<aui:a href="<%= kbArticleViewDisplayContext.getChildKBArticlesURLString(currentURL, kbArticle) %>">
+													<liferay-ui:message arguments="<%= kbArticleViewDisplayContext.getChildKBArticlesCount(scopeGroupId, kbArticle) %>" key="x-child-articles" />
 												</aui:a>
 											</span>
 										</c:if>
 
-										<span class="text-default">
-											<aui:workflow-status helpMessage='<%= kbArticle.isExpired() ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' markupView="lexicon" showHelpMessage="<%= kbArticle.isExpired() %>" showIcon="<%= false %>" showLabel="<%= false %>" status="<%= kbArticle.getStatus() %>" />
-										</span>
-									</liferay-ui:search-container-column-text>
+										<div>
```

Comment:
Is this new `<div>` necessary? Otherwise is better to remove it

Reply:
I'll checkit

Thanks!

Reply:
with :
```
<span class="text-default">
	<aui:workflow-status helpMessage='<%= kbArticle.isExpired() ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' markupView="lexicon" showHelpMessage="<%= kbArticle.isExpired() %>" showIcon="<%= false %>" showLabel="<%= false %>" status="<%= kbArticle.getStatus() %>" />
</span>
<c:if test="<%= kbArticleViewDisplayContext.isExpiringSoon(kbArticle) %>">
	<span class="label label-warning">
		<span class="label-item label-item-expand"><liferay-ui:message key="expiring-soon" /></span>
	</span>
	<liferay-ui:icon-help message='<%= kbArticle.getExpirationDate()!= null ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' />
</c:if>
```
<img width="1290" alt="image" src="https://user-images.githubusercontent.com/47524751/213406178-7b273160-4755-4656-8df0-2ec485f58590.png">


but with :
```
<span class="text-default">
	<aui:workflow-status helpMessage='<%= kbArticle.isExpired() ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' markupView="lexicon" showHelpMessage="<%= kbArticle.isExpired() %>" showIcon="<%= false %>" showLabel="<%= false %>" status="<%= kbArticle.getStatus() %>" />

	<c:if test="<%= kbArticleViewDisplayContext.isExpiringSoon(kbArticle) %>">
		<span class="label label-warning">
			<span class="label-item label-item-expand"><liferay-ui:message key="expiring-soon" /></span>
		</span>

		<liferay-ui:icon-help message='<%= kbArticle.getExpirationDate()!= null ? dateFormatDateTime.format(kbArticle.getExpirationDate()) : "" %>' />
	</c:if>
</span>
```
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/47524751/213409136-15884866-4385-4e94-bd9e-032b4b25fa9c.png">


So I will remove the div, and put all on the span


I'll rebase with master and push-f


thanks!!!